homeassistant/components/sensor/aftership.py

```diff
@@ -0,0 +1,161 @@
+"""
+Sensor for AfterShip.
+
+Gives a sensor of all non-delivered packages recorded in AfterShip.
+
+Requires an api key which can be aquired from
+https://secure.aftership.com/#/settings/api.
+
+Example configuration:
+
+sensor:
+   - platform: aftership
+     api_key: AFTERSHIP_API_KEY
+"""
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import (DOMAIN, PLATFORM_SCHEMA)
+
+from homeassistant.const import (
+    ATTR_ATTRIBUTION, CONF_API_KEY, CONF_NAME)
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+from homeassistant.exceptions import PlatformNotReady
+
+REQUIREMENTS = ['pyaftership==0.0.5']
```

Comment:
It looks like that the latest release is `0.1.1`.

Reply:
yeah, new update came out based on this PR - pushed update using latest that is also async. 

but just noticed the service is failing for this version so looking into that now.

Reply:
removed the service code to make the initial PR simpler.